.circleci/config.yml

```
@@ -265,13 +267,39 @@ jobs:
             bin/smoke_test --remote --no-source-env
       - notify-slack-smoke-test-status
       - store-smoketest-results
+  check-pinpoint-config:
+    docker:
+      - image: circleci/ruby:2.6-node-browsers
+    steps:
+      - checkout
+      - bundle-yarn-install
+      - run:
+          name: Check current AWS Pinpoint country support
+          command: |-
+            diff <(bundle exec ./scripts/pinpoint-supported-countries) config/country_dialing_codes.yml
+      - slack/status:
+          fail_only: true
+          failure_message: ":aws-emoji: :red_circle: AWS Pinpoint country configuration is out of date"
+
 workflows:
   version: 2
   release:
     jobs:
       - build
       - lints
 
+  daily-external-pinpoint-checker:
+    jobs:
+      - check-pinpoint-config
+    triggers:
+      - schedule:
+          # Once a day at 12pm
+          cron: "0 12 * * *"
+          filters:
+            branches:
+              only: master
+              ignore: master # Temporarily disabled due to manual phone support overrides.
```

```suggestion
              only: main
              ignore: main # Temporarily disabled due to manual phone support overrides.
```
we don't use that branch anymore